css/styles.css

```diff
@@ -0,0 +1,176 @@
+* {
+  margin: 0 0 0 0;
+}
+
+footer {
+  display: flex;
+  box-sizing: border-box;
+  height: 60px;
+  padding: 20px 0 0 20px;
+  margin: 0 0 0 0;
+  width: 100%;
+}
+
+footer h6 {
+  color: #6e6e6e;
+  font-weight: lighter;
+  flex: 1;
+}
+
+footer h6 span a {
+  text-decoration: none;
+  color: #00a8e3;
+}
+
+.link-item a {
+  text-decoration: none;
+  color: #00a8e3;
+  font-size: 0.8rem;
+  font-size: 0.6rem;
+}
+
+footer h6 span a:hover {
+  text-decoration: underline;
+}
+
+.main {
+  box-sizing: border-box;
+  width: 1300px;
+  height: 700px;
+  padding: 0 0 0 0;
+  margin-left: 30px;
+  margin-right: auto;
+  border: 1px solid #757575;
+}
+
+.main-section h1 {
+  position: absolute;
+  color: white;
+  font-size: 5rem;
+  font-family: 'Montserrat', sans-serif;
+  top: 250px;
+  left: 200px;
+}
+
+.main-section p {
+  font-size: 2rem;
+  line-height: 1;
+  font-family: 'Montserrat', sans-serif;
+  color: white;
+  position: absolute;
+  top: 350px;
+  left: 150px;
+}
+
+.main-section p span {
+  position: relative;
+  left: 50px;
+}
+
+.main-section p span span {
+  font-size: 1.7rem;
+}
+
+.nav-bar {
+  padding-left: 0;
+  padding-right: 20px;
+  position: absolute;
+  top: 160px;
+  right: 55px;
+  width: 90%;
+  display: flex;
+  background: linear-gradient(to top, #757575, #5f635f, #7d7d7d, #7c807c);
+  height: 40px;
+  justify-content: space-between;
+  text-align: center;
+  border-radius: 5px;
+}
```

Comment:
You could align the navbar to center of the screen, the issue you are having is that your navbar has an absolute position so it will be displayed in a different position depending on the screen.
first, remove the `right` property and replace it with the following
```
left: 50%;
transform: translateX(-50%);
````

Reply:
I dont know what are you talking about , you are not being clear with your request i have to guess what im supposed to do , my main image is already centered, the nav bar is flexible the nav Will not move around the page, don't try to request things that arent required to pass the review, i already made the previous changes you asked yesterday, and now you are asking me to change other elements on my page that you not stated before.

Reply:
@mekinus to pass the review the page should look like the original page, this is the same change I requested before, the image isn't centered and the navbar isn't in the center, the comment I made was a hint on how to fix that on all screens.

The issue here is that you used position absolute so the navbar will stay in that specific position while the other elements will change depending on the screen size.

Reply:
The navbar Will shrink and stretch along with the page size, the navbar is at the center like the archive is showing, the image is at center, there is no Space to move the image , I still don't know what vou are talking about . You are asking different things each review you make , and as result slowing my progress in the Curriculum

Reply:
This is how the original page looks like: 
<img width="1668" alt="Screen Shot 2019-11-12 at 7 39 11 PM" src="https://user-images.githubusercontent.com/33462180/68722681-39b1d880-0584-11ea-9af3-3abb337bfbcf.png">

And this is how your page is actually looking:
<img width="1671" alt="Screen Shot 2019-11-12 at 7 40 36 PM" src="https://user-images.githubusercontent.com/33462180/68722723-5fd77880-0584-11ea-8611-971fe26940f4.png">

You are using a position absolute for the navbar is not about the size but about the **position**, and this is what is causing this behavior, I gave you the cause on how to fix that in the comment above, please fix it and submit for a re-review, I'm not asking you for things that aren't required 😃

Reply:
My page is not looking like that to me , its what im telling you , my page looks pretty normal. Like the original site

Reply:
![image](https://user-images.githubusercontent.com/54003612/68723932-e007d900-0598-11ea-89c0-5071f38e72ff.png)